pyproject.toml

```diff
@@ -1,3 +1,73 @@
+[project]
+name = 'scylla-cluster-tests'
+version = '0.1'
+requires-python = ">=3.10, <3.14"
+dependencies = [
+    "aexpect==1.6.2",
+    "anyconfig==0.12.0",
+    "awscli==1.29.4",
+    "azure-identity==1.16.1",
+    "azure-mgmt-compute==33.1.0",
+    "azure-mgmt-network==28.1.0",
+    "azure-mgmt-resource==23.2.0",
+    "azure-mgmt-resourcegraph==8.0.0",
+    "azure-mgmt-subscription==3.1.1",
+    "azure-storage-blob==12.16.0",
+    "boto3-stubs[s3,ec2,dynamodb,pricing]==1.28.4",
+    "boto3==1.28.4",
+    "click==8.1.3",
+    "confluent-kafka==2.5.3",
+    "deepdiff==8.1.1",
+    "docker==7.1.0",
+    "elasticsearch==8.17.0",
+    "fabric==3.2.2",
+    "fastavro==1.9.7",
+    "gimme-aws-creds==2.8.2",
+    "google-api-python-client==2.93.0",
+    "google-cloud-compute==1.13.0",
+    "google-cloud-storage==2.10.0",
+    "hdrhistogram==0.9.2",
+    "invoke==2.2.0",
+    "jinja2==3.1.6",
+    "kubernetes==31.0.0",
+    "ldap3==2.9.1",
+    "moto[server]==5.1.1",
+    "mysql-connector-python==9.1.0",
+    "packaging>=24",
+    "paramiko==3.5.0",
+    "parameterized==0.8.1",
+    "pbr==6.1.0",
+    "pre-commit==4.0.1",
+    "prettytable==3.10.2",
+    "prometheus_client==0.11.0",
+    "pycryptodome==3.19.1",
+    "pydantic==2.10.5",
+    "PyGithub==2.1.1",
+    "pytest-random-order==1.0.4",
+    "pytest==8.3.4",
+    "python-jenkins==1.7.0",
+    "PyYAML==6.0.1",
+    "pyzmq==26.2.0",
+    "questionary==2.1.0",
+    "requests==2.32.2",
+    "scylla-driver==3.29.3",
+    "sortedcontainers==2.4.0",
+    "ssh2-python3==1.11.1",
+    "tcconfig==0.29.1",
+    "tenacity==9.0.0",
+]
+[dependency-groups]
```

Comment:
Maybe we should use extras ? so it can be used with plain pip as well ?

Reply:
I believe pip supports this as well: https://ichard26.github.io/blog/2025/04/whats-new-in-pip-25.1/#key-features-